testing/vcs/CMakeLists.txt

```diff
@@ -287,7 +287,7 @@ cdat_add_test(vcs_test_taylor_2_quads
       "${PYTHON_EXECUTABLE}"
       ${cdat_SOURCE_DIR}/testing/vcs/test_vcs_show.py
       )
-    FOREACH(gm boxfill isofill isoline vector meshfill )
+    FOREACH(gm boxfill isofill isoline meshfill )
```

Comment:
yes since they do not work and are currently slightly different between mac and linux.

see: #653